src/components/MessageBody.tsx

```diff
@@ -46,20 +73,78 @@ const useStyles = makeStyles((theme) => ({
     '& tr:nth-child(even)': {
       backgroundColor: 'lightgray',
     },
+    '& img': {
+      maxWidth: '100%',
+    },
   },
 }));
 
 type Props = {
   messageBody: string;
 };
 
+const renderCode = ({
+  inline,
+  className: classNameInit,
+  children: codeContent,
+  ...props
+}: CodeProps): ReactElement => {
+  const match = /language-(\w+)/.exec(classNameInit || '');
+  return !inline && match ? (
+    <Highlight
+      {...defaultProps}
+      theme={vsLight}
+      code={String(codeContent).replace(/\n$/, '')}
+      language={match[1] as Language}
```

Comment:
Are we guaranteed that `match[1]` exists? What if match is null, and could the array have less than 2 elements?

Reply:
Yes, when the class defining the language is present and we have a match, it will contain at index 1 the matched language.